src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -1,7 +1,13 @@
-import { mxgraph } from 'ts-mxgraph';
+import { mxgraph, mxgraphFactory } from 'ts-mxgraph';
 import Shape from '../../model/bpmn/shape/Shape';
 import Edge from '../../model/bpmn/edge/Edge';
 import BpmnModel from '../../model/bpmn/BpmnModel';
+import ShapeBpmnElement from '../../model/bpmn/shape/ShapeBpmnElement';
+
+const { mxPoint } = mxgraphFactory({
```

Comment:
we need to discuss how to make this call only once -> use the service 
to make sure we operates on same mxGraph objects.
it should be separate PR I think

Reply:
Yes, I agree, there are a lot of place where this is called.
Let's do it in a separate PR